cmake/onnxruntime_hosting.cmake

```diff
@@ -28,12 +28,6 @@ target_include_directories(${PROJECT_NAME} PRIVATE
 
 target_link_libraries(${PROJECT_NAME} PRIVATE
         ${Boost_LIBRARIES}
-        ${PROVIDERS_MKLDNN}
-        ${MKLML_SHARED_LIB}
-        ${PROVIDERS_CUDA}
-        ${onnxruntime_tvm_libs}
-        ${onnxruntime_libs}
```

Comment:
NIT, you may want to keep this one and remove the rest of them.

Reply:
I tried this and ${onnxruntime_libs} doesn't give us any of those libs. In fact, I can't find where it is set.